### DISCUSSION TITLE
it/blog/introduction-to-astro-a-new-front-end-framework

### DISCUSSION THREAD

**Discussion by giscus** [majesticooss/astros#20] [Blog]
# it/blog/introduction-to-astro-a-new-front-end-framework

Astros is a starter website template for Astro built with TailwindCSS and alpine js.

http://173.232.195.175/it/blog/introduction-to-astro-a-new-front-end-framework

**Comment by mkantowski** [comment]
helo

**Comment by mkantowski** [comment]
hello

**Comment by hananinas** [comment]
hej

**Reply by Ivacker** [reply]
WTF

**Comment by ThatMWCoder** [comment]
wow
